libs/dh/admin/shell/src/lib/dh-admin-shell.routes.ts

```diff
@@ -27,13 +31,24 @@ export const routes: Routes = [
   },
   {
     path: dhAdminUserManagementPath,
-    loadComponent: () =>
-      import('@energinet-datahub/dh/admin/feature-user-management').then(
-        (m) => m.DhAdminFeatureUserManagementComponent
-      ),
-    canActivate: [PermissionGuard(['users:manage'])],
-    data: {
-      titleTranslationKey: 'admin.userManagement.topBarTitle',
-    },
+    children: [
+      {
+        path: '',
+        pathMatch: 'full',
+        loadComponent: () =>
+          import('@energinet-datahub/dh/admin/feature-user-management').then(
+            (m) => m.DhAdminFeatureUserManagementComponent
+          ),
+        canActivate: [PermissionGuard(['users:manage'])],
+        data: {
+          titleTranslationKey: 'admin.userManagement.topBarTitle',
+        },
+      },
+      {
+        path: dhAdminUserRoleManagementCreatePath,
+        canActivate: [PermissionGuard(['users:manage'])],
+        component: DhAdminCreateUserRoleComponent,
```

Comment:
suggestion: this component can be lazy-loaded just like the one above. So the browser will only download the related code if the user navigates to `create-userrole` page.

Reply:
ok, will implement

Reply:
Fixed